master/internal/api/apiutils/utils.go

```diff
@@ -51,6 +53,7 @@ var (
 
 // MapAndFilterErrors takes in an error at the db level and translates it into a standard error.
 func MapAndFilterErrors(err error, passthrough map[error]bool, mapping map[error]error) error {
+	// CHECK: should this be in api/error.go?
```

Comment:
We can talk about this when we all get back, should be an easy change using goland's refactor tools if we want to move it.